app/code/Magento/Downloadable/Helper/Catalog/Product/Configuration.php

```diff
@@ -64,13 +67,12 @@ public function getLinks(\Magento\Catalog\Model\Product\Configuration\Item\ItemI
     public function getLinksTitle($product)
     {
         $title = $product->getLinksTitle();
-        if (strlen($title)) {
-            return $title;
+
+        if (!$title) {
+            $title = $this->scopeConfig->getValue(Link::XML_PATH_LINKS_TITLE, ScopeInterface::SCOPE_STORE);
```

Comment:
Hello @anzin 
This fix has already delivered by [this PR](https://github.com/magento/magento2/pull/34689/files)
Please roll back this file